src/Microsoft.Framework.Configuration.Binder/ConfigurationBinder.cs

```diff
@@ -25,20 +25,30 @@ public static void Bind(this IConfiguration configuration, object instance)
             }
         }
 
-        public static T Get<T>(this IConfiguration configuration)
+        public static T Get<T>(this IConfiguration configuration, T defaultValue)
         {
-            var value = Get(configuration, typeof(T));
+            var value = configuration.Get(typeof(T));
             if (value == null)
```

Comment:
Let's go with this behavior for now. I think it is probably not what we want but I just discovered that things are more broken than I though downstream in https://github.com/aspnet/Configuration/pull/309/files?diff=split#diff-ad58cb8470446e5e357497cee03e8e19R114. That is why I need to do the exploratory testing :smile: